sentence_transformers/util.py

```diff
@@ -361,7 +361,7 @@ def community_detection(embeddings, threshold=0.75, min_community_size=10, batch
     min_community_size = min(min_community_size, len(embeddings))
     sort_max_size = min(max(2 * min_community_size, 50), len(embeddings))
 
-    for start_idx in range(0, len(embeddings), batch_size):
+    for start_idx in tqdm(range(0, len(embeddings), batch_size), desc="Find Cluster", disable=not show_progress_bar):
```

Comment:
```suggestion
    for start_idx in tqdm(range(0, len(embeddings), batch_size), desc="Finding clusters", disable=not show_progress_bar):
```
I think this might be a bit clearer, next to the progress bar. Other than that, this looks great!